tests/regression/roles/sonic_ospfv3_area/defaults/main.yml

```diff
@@ -0,0 +1,302 @@
+    - "ipv6 prefix-list pf2 seq 10 permit 1::2/64"
+
+sonic_ospfv3_area_tests:
+- name: test_case_01
+  description: Create ospfv3 area
+  state: merged
+  input:
+    - area_id: 1
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf1}}"
+    - area_id: 2
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf1}}"
+      stub:
+        enabled: true
+        no_summary: true
+    - area_id: 3
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf1}}"
+      filter_list_in: pf1
+      filter_list_out: pf2
+      ranges:
+        - prefix: 1::1/64
+          advertise: true
+          cost: 4
+        - prefix: 2::2/64
+          advertise: false
+        - prefix: 3::3/64
+          advertise: true
+          cost: 10
+    - area_id: 4
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf2}}"
+      nssa:
+        enabled: true
+        default_originate:
+          enabled: true
+          metric: 10
+          metric_type: 1
+    - area_id: 5
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf2}}"
+      nssa:
+        enabled: true
+        no_summary: true
+
+- name: test_case_02
+  description: Add additional ospfv3 area details
+  state: merged
+  input:
+    - area_id: 1
+      vrf_name: "default"
+      filter_list_in: pf1
+      filter_list_out: pf2
+      ranges:
+        - prefix: 1::1/64
+          advertise: true
+          cost: 4
+        - prefix: 2::2/64
+          advertise: false
+        - prefix: 3::3/64
+          advertise: true
+          cost: 10
+    - area_id: 3
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf1}}"
+      filter_list_in: pf1
+      filter_list_out: pf2
+      ranges:
+        - prefix: 1::1/64
+          advertise: true
+          cost: 4
+        - prefix: 2::2/64
+          advertise: false
+    - area_id: 4
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf2}}"
+      nssa:
+        enabled: true
+        default_originate:
+          enabled: true
+          metric: 15
+          metric_type: 2
+    - area_id: 5
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf2}}"
+      nssa:
+        enabled: true
+        no_summary: false
+    - area_id: 5
+      vrf_name: "default"
+      stub:
+        enabled: true
+        no_summary: true
+
+- name: test_case_03
+  description: Create more ospfv3 areas.
+  state: merged
+  input:
+    - vrf_name: "default"
+      area_id: 10
+      ranges:
+        - prefix: 20::1/64
+          advertise: true
+          cost: 15
+        - prefix: 21::1/64
+          advertise: true
+          cost: 20
+    - vrf_name: "{{sonic_ospfv3_area_test_vars.vrf2}}"
+      area_id: 3
+      nssa:
+        enabled: true
+        default_originate:
+          enabled: true
+          metric_type: 1
+          metric: 10
+
+- name: test_case_del_01
+  description: Delete subsections of ospfv3 area config
+  state: deleted
+  input:
+    - area_id: 4
+      vrf_name: "{{sonic_ospfv3_area_test_vars.vrf2}}"
+      nssa:
+        enabled: false
```

Comment:
The configured "enabled" value before this was "true". Why is this set to "false"? (It gets deleted anyway, which is not correct behavior.)

Reply:
Just wanted to test the behavior over "false" value.
Changed it to "true"